tests/recovery.test.ts

```diff
@@ -93,21 +93,16 @@ describe("recovery-tests", () => {
       expect(LocalRecovery.recoveryCount).toBeLessThanOrEqual(LocalRecovery.maxRecoveryAttempts);
     }
 
-    for (let i = 0; i < LocalRecovery.maxRecoveryAttempts * 2; i++) {
-      await testRuntime.startWorkflow(LocalRecovery, handle.getWorkflowUUID()).deadLetterWorkflow();
-      expect(LocalRecovery.recoveryCount).toBe(i + LocalRecovery.maxRecoveryAttempts + 1);
-    }
```

Comment:
This path does not make sense anymore (you can't call a workflow in "non recovery mode")